### PR TITLE
Fixed agent and agency routes to be more consistent and restful

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: Api/Controllers/AgentControllers/AgentsController.cs [hunk 1]
```diff
@@ -404,7 +404,7 @@ private async Task<string> GetUserEmail()
         /// <summary>
         ///     Disables a given agent to operate using a given agency
         /// </summary>
-        [HttpPost("agents/{agentId}/disable")]
+        [HttpPost("agency/agents/{agentId}/disable")]
```
Comment (hasmikharutyunyan) [inline Api/Controllers/AgentControllers/AgentsController.cs:407]: why we need `agency`  here ? It is an agentController and I think it is ok to not use `ageny`

**Review by hasmikharutyunyan — COMMENTED** [review]
maybe needed to change `AgentsController` to `AgentController`?

File: Api/Controllers/AgentControllers/AgentsController.cs (hunk 1 above)
Comment (hasmikharutyunyan) [reply Api/Controllers/AgentControllers/AgentsController.cs:407]: Or replace this action into `AgencyController`

Comment (mingazhev) [reply Api/Controllers/AgentControllers/AgentsController.cs:407]: Maybe, I did not move methods there, tried to change only the routes to make code review easier.
I'm agree to move agency methods to the separate controllers, but I think routes will be the same

**Comment by mingazhev** [comment]
> 
> 
> maybe needed to change `AgentsController` to `AgentController`?

Sure, I'll change later. Just tried to make PR as lightweight as possible

File: Api/Controllers/AgentControllers/AgentsController.cs (hunk 1 above)
Comment (hasmikharutyunyan) [reply Api/Controllers/AgentControllers/AgentsController.cs:407]: So, It is will be done in the next PR-s, I can approve this?

Comment (mingazhev) [reply Api/Controllers/AgentControllers/AgentsController.cs:407]: Yes, if you are agree with the routes

**Review by hasmikharutyunyan — APPROVED** [review]

**Review by alexander-plekhanov — APPROVED** [review]

**Pull request merged**